docs/README.md

```diff
@@ -0,0 +1,29 @@
+# How To Contribute
+## Introduction
+The NGINX Ingress Controller makes use of the [Hugo](https://gohugo.io/) static site generator.
+
+Documentation is stored within the `docs/`subfolder of the main repository, and is where `hugo` commands should be run.
+
+We use a feature branch workflow: changes should be made from a branch originating from `main`.
+
+Commits should be squashed, and pull requests (PRs) should then target the `main` branch.
+
+Once a PR has been accepted, its changes should be cherry picked onto the release branch.
+
+## Set-up
+**Requirements**
+* [git](https://git-scm.com/downloads)
+* [hugo](https://gohugo.io/installation/)
+
+**Quick Start**
+```
+git clone git@github.com:nginxinc/kubernetes-ingress.git
+cd docs/
+hugo server -e production -b "http://127.0.0.1"
```

Comment:
It shouldn't be necessary to use the production environment when running hugo locally (needs verification, though). You should be able to just run `hugo server`. (Using the production environment inserts `docs.nginx.com` as the baseUrl instead of just using localhost.)